{
	"id": "oval:org.debian:def:199383274605541855427100625255498474767",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2019-20907 python3.9",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 11",
			"product": "python3.9"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2019-20907",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-20907"
			}
		],
		"description": "In Lib/tarfile.py in Python through 3.8.3, an attacker is able to craft a TAR archive leading to an infinite loop when opened by tarfile.open, because _proc_pax lacks header validation.",
		"debian": {}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:17159",
								"comment": "python3.9 DPKG is earlier than 3.9.0~b5-1"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 11 is installed"
			}
		]
	}
}
